client/reader/search-stream/site-results.jsx

```diff
@@ -82,11 +82,14 @@ export default connect(
 		} );
 
 		// Check if searchResults has one item and if it has a feed_ID
-		if ( searchResults && searchResults.length === 1 ) {
+		if ( searchResults && searchResults.length > 0 ) {
```

Comment:
In general i feel like we are making some very subtle changes in these files. This seems like the most substantial? Is this where we think the regression happened: a change to the backend resulting in more than 1 result in these cases causing this to get bypassed?

Also note the comment above could be updated now.